apps/daimo-mobile/src/action/useCreateAccount.ts

```diff
@@ -24,7 +24,7 @@ export function useCreateAccount(name: string): ActHandle {
   const result = rpcHook.deployWallet.useMutation();
   const exec = async () => {
     if (!pubKeyHex) return;
-    setAS("loading", "Deploying contract...");
```

Comment:
this is user facing, so i'd still call it an account

related: "the user doesn't know what a key is" from this great guide

https://github.com/signalapp/Signal-Android/blob/main/CONTRIBUTING.md